.eslintrc.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
     "max-len": ["error", { "code": 110 }],
     "no-param-reassign": ["error", { "props": false }],
     "operator-linebreak": ["error", "before"],
-    "prefer-regex-literals": "off",
+    "prefer-regex-literals": "error",
```

Comment:
Since this is fixed by your PR, we can safely remove this line entirely and allow it to be included by default from the airbnb config